clipping_manager/views.py

```diff
@@ -50,20 +49,42 @@ def get_context_data(self, **kwargs):
         return ctx
 
     def get_queryset(self):
-        qs = Clipping.objects.select_related('book').for_user(user=self.request.user)
+        qs = Clipping.objects.select_related('book').for_user(user=self.request.user) \
+                                                    .exclude(deleted = True)
```

Comment:
Seeing how often you had to add `exclude(deleted=True)` it might make sense to override the default manager to always exclude it by default (and add a `with_deleted` or something like thatto the manager)